nri-config-generator/main.go

```diff
@@ -67,7 +68,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	exporterName := fmt.Sprintf("%s-exporter", integration)
+	exporterName := getExporterNameFromIntegration(integration)
```

Comment:
no, it doesn't, on line 101:
	vars[varIntegrationName] = integration 
it has the nri-integration, because in common_functions.sh there is :
 export PACKAGE_NAME=nri-${NAME}
 
 But, there is the exporteed env var NAME that has it, I could modify the compile step on the make and the code to get that name variable and use it here, what do you think? I ilike it since it's more clean.